.github/workflow_templates/on-pull-request-labeled.yml

```diff
@@ -4,19 +4,21 @@
 name: Rerun workflows for pull request
 
 on:
-  pull_request:
-    types: [labeled,unlabeled]
-
+  pull_request_target:
+    types: [labeled, unlabeled]
 jobs:
+{!{ tmpl.Exec "pull_request_info" . | strings.Indent 2 }!}
   rerun_workflow_for_pull_request:
     name: Rerun workflow for pull request
     runs-on: ubuntu-latest
+    needs:
+      - pull_request_info
     steps:
-{!{ tmpl.Exec "checkout_step" . | strings.Indent 6 }!}
```

Comment:
This step is required for `const ci = require('./.github/scripts/js/ci');`

Reply:
I thought it can be achieved by `pull_request_info`. It has all the same as in main for external PRs and possibly updated code for our own Prs

Reply:
returned back